libevmasm/PeepholeOptimiser.cpp

```diff
@@ -173,6 +173,32 @@ struct CommutativeSwap: SimplePeepholeOptimizerMethod<CommutativeSwap, 2>
 	}
 };
 
+struct SwapComparison: SimplePeepholeOptimizerMethod<SwapComparison, 2>
+{
+	static bool applySimple(AssemblyItem const& _swap, AssemblyItem const& _op, std::back_insert_iterator<AssemblyItems> _out)
+	{
+		map<Instruction, Instruction> swappableOps{
```

Comment:
Sometimes you also have to do things like
```
swappableOps = map<Instruction, Instruction>{...};
```
i.e. name the type in front of the braces.